### PR TITLE
make datetime type fields be serialized as datetime and in utc

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/contacts/models.py [hunk 1]
```diff
@@ -241,19 +242,19 @@ def get_field_raw(self, key):
         value = self.get_field(key)
         return value.string_value if value else None
 
-    def get_field_display(self, key):
+    def get_field_display(self, key, normalize_datetime=False):
```
Comment (nicpottier) [inline temba/contacts/models.py]: This seems a bit goofy.. this isn't really a display value anymore in this case, so passing in normalize_datetime seems odd.

Would rather see a method specific for serialization that takes care of this.

Comment (rowanseymour) [inline temba/contacts/models.py]: Yeah let's keep things separate here - `get_field_display` is for display. For example we might have a DECIMAL field which is 33.33333333 but we display it in the UI as "33.33".

So we could have the API expose more complete serialised values. ISO8601 for dates, all decimal places for decimals etc. For dates we already have `datetime_to_json_date` for consistent formatting  which already does conversion to UTC.

Comment (norkans7) [inline temba/contacts/models.py]: Ok

**Comment by nicpottier** [comment]
Looks good.

**Pull request merged**